rca/wagtailapi/filters.py

```diff
@@ -16,9 +16,9 @@ def filter_queryset(self, request, queryset, view):
         pks = request.GET.getlist("project", [])
 
         if pks:
-            queryset = queryset.filter(degree_level__in=pks).order_by("title").live()
+            queryset = queryset.filter(degree_level__in=pks)
 
-        return queryset
+        return queryset.order_by("title").live()
```

Comment:
Good work finding this bug! Not a blocker, but a suggestion: I can see the same pattern on `SubjectsFilter` and `RelatedSchoolsFilter` below, and wondering if they ought to be changed as well.

Reply:
I'm not aware that the other filters are affected by the same bug so I'm going to leave it for now, thanks for spotting!